borg/archiver.py

```diff
@@ -177,6 +178,7 @@ def do_check(self, args, repository):
     @with_repository()
     def do_change_passphrase(self, args, repository, manifest, key):
         """Change repository key file passphrase"""
+        check_repository_compatibility(manifest, REPO_OPERATION_CHECK)
```

Comment:
I was unsure how to make that nice.
- There are places where the manifest is loaded only later. So it can not really be universal
- should this be additional in the exiting decorators or would it be stacked?
- I really want this to be in 1.1, so i do consider it ok if it needs to be optimized later.

Maybe we can just load the manifest more often centrally that would certainly make this easier as decorator?